anthos-bm-gcp-terraform/main.tf

```diff
@@ -323,6 +323,16 @@ module "init_hosts" {
   cluster_yaml_path      = local.cluster_yaml_file
 }
 
+module "gke_hub_membership" {
+  source   = "terraform-google-modules/gcloud/google"
+  version  = "~>3.1.1"
+  enabled  = var.mode == "install" || var.mode == "manuallb" ? true : false
```

Comment:
I think we would want it to be enabled in any mode. Even with the normal mode, the user might log in and run the ABM installation. Can we have it always enabled.

What happens if there is no membership? (i.e. User just creates the GCE VMs but doesn't install ABM). Just this silently fail or will the terraform destroy fail completely?

Reply:
Let me test it out, if we can have it silently fail I think that would be ideal for this use case.

Reply:
OK - I've modified so it will always "succeed", so it won't block the TF destroy.